example/spec/helpers/testUtils.js

```diff
@@ -100,7 +101,8 @@ function updateAndUploadTestFileToBucket(file, bucket, prefix = 'cumulus-test-da
   return s3().putObject({
     Bucket: bucket,
     Key: `${prefix}/${key}`,
-    Body: data
+    Body: data,
+    ContentType: mime.lookup(key) || null
```

Comment:
Can we just use the helper from the ingest package here? Improves consistency and you can remove the dependency of the `example/` directory on `mime-types`

Reply:
If we do this, then we have to pull `@cumulus/ingest` into the `example/` directory. It's kind of "half a dozen in one and six in the other" to me.

Reply:
Ah, I thought it was already depending on `@cumulus/ingest`. That's fine then. But in any case we need the updated lockfile for example/ committed